### ISSUE TITLE
Authors names in the reference should be displayed as First Middle Last, First Middle Last, and First Middle Last

### ISSUE THREAD

**Issue by njarboe** [earthref/MagIC#152]
Authors names in the reference should be displayed as First Middle Last, First Middle Last, and First Middle Last.
It looks like journals are using this style and it makes parsing each name much easier than:
Last, First Middle, Last, First, Last, First Middle, and Last, First.

**Comment by rminnett** [comment]
Yes, I agree - this is a convention that Anthony used over a decade ago when the Reference Database was created for EarthRef.
